src/helpers/index.tsx

```diff
@@ -306,6 +308,53 @@ export const handleContractError = (e: any) => {
   if (EnvHelper.env.NODE_ENV !== "production") console.warn("caught error in slices; usually network related", e);
 };
 
+/**
+ * Converts a supported networkId to its hexadecimal value as a string
+ */
+export const networkIdToHexString = (id: NetworkId) => "0x" + id.toString(16);
+
+/**
+ * Determines if app is viewed within an <iframe></iframe>
+ */
+export const isIFrame = () => window.location !== window.parent.location;
+
+/**
+ * Assertion function helpful for asserting `enabled`
+ * values from within a `react-query` function.
+ * @param value The value(s) to assert
+ * @param queryKey Key of current query
+ */
+export function queryAssertion(value: unknown, queryKey: any = "not specified"): asserts value {
+  if (!value) throw new Error(`Failed react-query assertion for key: ${queryKey}`);
+}
+
+/**
+ * Assertion function
+ */
+export function assert(value: unknown, message: string | Error): asserts value {
+  if (!value) throw message instanceof Error ? message : new Error(message);
+}
+
+/**
+ * Converts gOHM to OHM. Mimics `balanceFrom()` gOHM contract function.
+ * @returns Formatted string representation of OHM equivalent.
+ */
+export const convertGohmToOhm = (amount: BigNumber, index: BigNumber): string => {
+  return formatUnits(amount.div(10 ** 9).mul(index), 36);
```

Comment:
Can refactor these magic numbers as constants

Reply:
agree

Reply:
I'll be honest here, these two decimals numbers in the `convertGohmToOhm` and `convertOhmToGohm` functions are magical. I'm not really sure why it needs to be 36 and 18 respectively, just that those are the correct values to put the decimal in the right spot. I'll try look into why that's the case tomorrow and update you with what I find.

Reply:
Essentially, it's do with how BigNumber handles math operations internally. It's weird here because we are converting between gohm (an 18 decimal number) and ohm (only a 9 decimal number), so it adds padding to ensure we don't lose precision when dividing/multiplying between the two.

Reply:
Is it alrght if I leave these specific two numbers as is? I feel like it doesn't really make sense to make these their own constants, but up to you guys @mrunbanked @appleseed-iii

Reply:
yea I think that's fine